lang/gcc14/Portfile

```diff
@@ -12,7 +12,7 @@ name                gcc${gcc_major_v}
 
 homepage            https://gcc.gnu.org/
 
-platforms           {darwin >= 10}
+platforms           {darwin >= 8}
```

Comment:
While this is perhaps factually correct, it is redundant, IMO, since MacPorts does not support 10.3 for years. I.e. dependencies won’t build. Just delete `platforms` line.

Reply:
Done